pkg/core/pipeline/resource/main.go

```diff
@@ -118,6 +121,7 @@ func GetResourceMapping() map[string]interface{} {
 
 	return map[string]interface{}{
 		"aws/ami":       &awsami.Spec{},
+		"cargopackage":  &cargopackage.Spec{},
```

Comment:
I am wondering if it's not better to just use "cargo" instead of "cargopackage"
@hervelemeur is more picky than me on this :)

Reply:
I'm open to change, I used `cargopackage` to be consistent with `dockerdigest`, `dockerfile`, `dockerimage`, ...